### ISSUE TITLE
Play as a team

### ISSUE THREAD

**Issue by FLemon** [FLemon/FLemon.github.io#2]
# Something about team play
## tl;dr

When I am in a team, I play with the following principles(at least i try to):

**Be Open**: before decision had been made, try hard to reason with team with what I am against, and get ready to be approved wrong, and accept the fact.

**Be Modest**: accept the fact that I am not superman, nor god, nor advance evolved alien(wait, am i?), so try to understand those who supports what I am up against.

**Be Proactive**: take every suggestion seriously like my own, and speak up my thoughts so that the team can hear your concern.

**Be Supportive**: decision made towards other's suggestion, support the team in all ways I can contribute to, from strategy planning to tea making.

**Be Unity**: anything goes wrong, I am happy to share the work load to fix the problem, share the blame, and swallow what I might had in mind about 'I told you so'.

**Be Respectful**: respect team members even when they make a mistake, big or small, but don't expect they should do the same to you.

**Be Grateful**: when outcome is positive, celebrate with team, cheer for the team, and absorb the successful experience; when outcome is negative, help team to collect information/data to learn the lesson, console the team, and absorb the failure experience.

**Be Responsive**: decision made towards my suggestion, try hard to foreseen the risks and impact of them, and get ready to stood up to take blame.

so, overall:
- either don't do it at all, or try the best to do it.
- rather spend time and effort to achieve a negative result, than spend time wasted on achieving nothing.
- stay and play well, or leave and watch them fall.

These are my summarises from what I've learnt after 10+ years of playing **M**assively **M**ultiplayer **O**nline **R**ole-**P**laying **G**ames
